contrib/win32/install/install/README.md

```diff
@@ -0,0 +1,63 @@
+# Windows OpenSSH - MSI project
+
+The project, based on the [WiX Toolset](https://wixtoolset.org/), creates a Windows MSI package to install OpenSSH on Windows computers.
+
+The projects supports building MSI packages for 32bit and 64bit platforms. In addition the MSI supports a wider range of installation flags to perform the installation of OpenSSH complete silently. Furthermore the MSI ensures that Debug end Release Versions cannot installed simultaneously. When the OpenSSH server components are installed, the MSI will also configure the Windows Firewall, to ensure that the SSH Server is reachable over the network.
+
+## Build Parameters
+
+The following parameters are supported 
+
+* `CONFIGURATION`: (required) the configuration used; Valid values: `Release`, `Debug`
+* `VERSION`: (required) the version tag of the MSI
+* `SOURCEDIR`: (required) the path to the input files
+* `MANUFACTURER`: (optional) the manufacturer of the MSI; default: Microsoft Corporation
+* `PRODUCTID`: (optional) the Product GUID of the created MSI; default: dynamic (`*`), changes every time the MSI is build
+* `IsPreview`: (optional) create a MSI for a previw version of OpenSSH, default: `false`
+
+The parameters are passed via `Msbuild` properties.
+
+Example:
+
+```shell
+$ msbuild openssh-install.wixproj /p:Version=8.1.1 /p:Configuration=Release /p:Platform=x64 /p:SourceDir=c:\openssh-portable\bin /p:ProductId="15512e42-e946-45f9-bbe5-a8e495bd2e2b"
+```
+
+## MSI internals
+
+### Upgrade Codes
+
+https://docs.microsoft.com/en-us/windows/win32/msi/using-an-upgradecode
+
+#### x64 (64-bit)
+
+* Preview version: `5a388d64-a109-4d85-bddc-7f46e1e1e520`
```

Comment:
What's the importance of these GUIDs? Given these are hardcoded, is there a need for changing them when a new OpenSSH version is published?